stable/nginx-ingress/templates/controller-psp.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if .Values.podSecurityPolicy.enabled}}
-apiVersion: extensions/v1beta1
+apiVersion: policy/v1beta1
```

Comment:
Same as previous comment.

> I think this could potentially be a breaking change for some users, so can we just make this configurable with the default being the existing API version?

Reply:
The **PodSecurityPolicy** resource is no longer available in `v1.16` under the `extensions/v1beta1` API. It is now only available in the `policy/v1beta1` API. This was available in the `policy/v1beta1` API since `v1.10`. This should be backward compatible, the API was just changed to a more stable endpoint.

Source: https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/